docs/clients/promtail/configuration.md

```diff
@@ -970,3 +1037,27 @@ scrape_configs:
       - source_labels: ['__journal__systemd_unit']
         target_label: 'unit'
 ```
+
+## Example Syslog Config
+
+```yaml
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://ip_or_hostname_where_loki_runns:3100/loki/api/v1/push
```

Comment:
Very small nit, but can we simplify the path here?

Maybe just `http://loki_address:3100/loki/api/v1/push`?